src/main/java/cr/pernix/dashboard/resources/CompanyResource.java

```diff
@@ -0,0 +1,56 @@
+package cr.pernix.dashboard.resources;
+
+import java.util.List;
+
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.GenericEntity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import cr.pernix.dashboard.models.Company;
+import cr.pernix.dashboard.services.CompanyService;
+
+@Path("company")
+public class CompanyResource {
+
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response get() throws Exception {
+        List<Company> result = CompanyService.getInstance().get();
+        GenericEntity<List<Company>> list = new GenericEntity<List<Company>>(result) {
+        };
+        return Response.ok(list).build();
+    }
+
+    @GET
+    @Path("/{id}")
+    public Response get(@PathParam("id") int id) throws Exception {
+        Company company = CompanyService.getInstance().get(id);
+        return Response.ok(company).build();
+    }
+
+    @POST
+    public Response create(Company company) throws Exception {
+        CompanyService.getInstance().saveOrUpdate(company);
+        return Response.ok().build();
+    }
+
+    @PUT
```

Comment:
look's fine for this entity, but be careful when you have relationships with other entities. In those cases I prefer to add a parameter Id to get the original instance, also, in a concurrency model you make sure you have the last version.